src/dendrogram.jl

```diff
@@ -1,4 +1,5 @@
-function treepositions(hc::Hclust, useheight::Bool)
+function treepositions(hc::Hclust, useheight::Bool, orientation=:v)
```

Comment:
In Plots.jl we use the symbols `:vertical` and `:horizontal` for `orientation

Reply:
Changed. Don't know anymore why I used :v and :h